src/config.rs

```diff
@@ -100,9 +125,9 @@ fn default_server_urls() -> Vec<String> {
     vec!["http://127.0.0.1:8080".to_string()]
 }
 
-fn default_pdb_config_section() -> PdbConfigSection {
+fn empty_pdb_config_section() -> PdbConfigSection {
     PdbConfigSection {
-        server_urls: Some(default_server_urls()),
+        server_urls: None,
```

Comment:
We already do this defaulting elsewhere and it just gets in the way here when trying to do fallback correctly (merging configs).